deploy/operator/config/system-config.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
 spec:
   builders:
     ubi8:
-      image: quay.io/redhat-appstudio/hacbs-jdk17-builder:f0c0dbbd5926e7a38af245b4e48a38ddd2d17359
-      tag: jdk:8;11;17,maven:3.8,gradle:8.0.2;7.4.2;6.9.2;5.6.4;4.10.3,sbt:1.8.0,ant:1.10.13
+      image: quay.io/redhat-appstudio/pull-request-builds:hacbs-jdk17-builder-1fa75e6c1ff4716b6dd100ba3346e0f25ffe771a
+      tag: jdk:8;11;17,maven:3.8.8;3.9.5,gradle:8.4;8.3;8.0.2;7.4.2;6.9.2,sbt:1.8.0,ant:1.10.13
```

Comment:
If tag is coming from jvm-build-service-builder-images via https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/operator/config/kustomization.yaml#L11 does tag need to be specified here as well? Or is this done for testing only?

Reply:
this is testing only